shacl/generate_shacl.py

```diff
@@ -40,7 +40,7 @@ def addRangeShape(propertyName, expectedType):
     G.add((BSH[shapename], A, SH.NodeShape))
     G.add((sh_prop, SH["path"], BRICK[propertyName]))
     G.add((sh_prop, SH["class"], expectedType))
-    G.add((BSH[shapename], SH.targetSubjectsOf, BRICK[propertyName]))
+    G.add((BSH[shapename], SH.targetObjectsOf, BRICK[propertyName]))
```

Comment:
I'm pretty sure this needs to be `targetSubjectsOf` because the `sh:property` property will target the objects of the target property (the property this SHACL shape is about) for us.

Reply:
```ttl
bsh:hasSiteRangeShape a sh:NodeShape ;
    sh:property [ sh:class brick:Site ;
            sh:message "Property hasSite has object with incorrect type" ;
            sh:path brick:hasSite ] ;
    sh:targetObjectsOf brick:hasSite .
```

This would be shape that was generated under the existing code. Because of `sh:targetObjectsof`, this shape applies to the `Site` instances! Which don't have `hasSite` properties.